proto/kvrpcpb.proto

```diff
@@ -198,6 +198,7 @@ message CheckTxnStatusRequest {
 }
 
 message CheckTxnStatusResponse {
+    reserved 6, 7;
```

Comment:
Why do we need to reserve 6 and 7, this proto has never been used in a published version, only on master, so it should be OK to make a breaking change.

Reply:
Reserving these fields helps us pass the integration tests.
If we just change the type of the field, when we first upgrade TiKV, the old version of TiDB will receive a proto it cannot handle. And vice versa. Then, we have to force merge a TiKV PR or TiDB PR.

Reply:
Can we remove them later? Seems better to force merge once, rather than live with cruft forever

Reply:
I didn't think it's cruft... Will reserved fields have extra cost? If so, we can force merge at first.

Reply:
I don't think it there is performance cost, but it is just a bit ugly to read. It is ok now, but if every proto has something like `reserved 6, 7, 10, 11, 12, 15, 16, 17;` or something

Reply:
Agree, it's ugly.

Reply:
ok.

Reply:
It's indeed a breaking change and forces BR to upgrade kvproto and TiDB https://github.com/pingcap/br/pull/481.

I suggest before we make any change to kvproto we should first make sure the feature development and test are almost complete.

Reply:
> I suggest before we make any change to kvproto we should first make sure the feature development and test are almost complete.

It seems like we should have some conventions, certianly